applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -605,6 +614,41 @@ See :ref:`app_build_file_suffixes` and :ref:`cmake_options` for more information
 
 .. _matter_bridge_testing:
 
+Configure the functionality of the Matter-Bridge device
+-------------------------------------------------------
+
+The device supports two configurations: basic and extended. 
+The basic functionality enables communication through Matter over Bluetooth LE. 
+Extended functionality includes basic functionality and configures the device to work as a smart plug.
+
+To use the basic functionality of the device, use the following command to build.
+
+``nrf54h20dk/nrf54h20/cpuapp``
+
+.. code-block:: console
+
+   west build -b nrf54h20dk/nrf54h20/cpuapp -p -- -DSB_CONFIG_WIFI_NRF700X=y -DCONFIG_CHIP_WIFI=y -Dmatter_bridge_SHIELD=nrf700x_nrf54h20dk -DCONFIG_BRIDGED_DEVICE_BT=y
+
+``nrf7002dk/nrf5340/cpuapp``
+
+.. code-block:: console
+
+   west build -b nrf7002dk/nrf5340/cpuapp -p -- -DCONFIG_BRIDGED_DEVICE_BT=y
+
+
+To enable the extended functionality, run the following command:
+
+.. code-block:: console
```

Comment:
The  ``nrf54h20dk/nrf54h20/cpuapp`` is missing before this one. Also I'm not sure how graphically it's going to look on the page after rendering. If you modify the docs, you will always see the following comment in your PR that includes a link to the rendered docs, so you can check how it's going to look like:

![image](https://github.com/user-attachments/assets/18aca0bd-856a-4a0c-bdb2-6bf85e26cfe0)